stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<p>"create expectation for the match up" -&gt; "create an expectation for the match up"?</p><p> </p><p>For the parameter key (and variable usage elsewhere) it would be nice to format it appropriately. Either <code>k</code> or $k$ depending on whether it's code or maths.</p><p> </p><p>This section is really nice and I love the way you're building your model up using new techniques. It would be really interesting to know whether Elo is considered a heuristic, or if it has a mathematical basis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Added both formatting and more information about Elo.</p><p> </p>

Reply:
<p>And fixed the typo!</p>